doc/filters/data_uri.rst

```diff
@@ -25,15 +25,15 @@ The ``data_uri`` filter generates a URL using the data scheme as defined in RFC
 
 .. note::
 
-    The ``data_uri`` filter is part of the ``HtmlExtension`` which is not
-    installed by default. Install it first:
+    The ``data_uri`` filter is part of the [``HtmlExtension``](https://github.com/twigphp/html-extra)
+    which is not installed by default. Install it first:
 
     .. code-block:: bash
 
         $ composer require twig/html-extra
 
-    Then, use the ``twig/extra-bundle`` on Symfony projects or add the extension
-    explicitly on the Twig environment::
+    On Symfony projects, install the [``twig/extra-bundle``](https://github.com/twigphp/twig-extra-bundle).
+    Otherwise, add the extension explicitly on the Twig environment::
```

Comment:
Same here

Reply:
Well, just noticing that it's not really the same here, since there are no installation instructions for Symfony users ;-) The link would cover that. What do you suggest?

Reply:
These are rst docs, the used syntax (markdown) doesn't work 👍🏼 

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

Reply:
Thanks :-)